ext/pdo_sqlite/tests/gh13991.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Fix GH-13984: Buffer size is now checked before memcmp
+--EXTENSIONS--
+pdo_sqlite
+--FILE--
+<?php
+$dbfile = 'z.db';
```

Comment:
I did this because specifying FullPath would make the string longer and not be overloaded.

Reply:
Hm right... This can clash with local files though, so perhaps it's better to use `13991` as filename?